templates/hash.description.html

```diff
@@ -0,0 +1,85 @@
+{% extends "index.html" %}
+{% block body %}
+
+<div class="row">
+   <div class="col-lg-12 col-md-12 col-sm-12">
+        <div class="panel panel-default">
+            <div class="panel-heading" id="head">
+                <div class="row">
+					<div class="col-lg-12 col-md-12 col-sm-12">
+						<b><span class="pozdrav1">Zgoščevalne funkcije</span></b>
+					</div>
+				</div>
+			</div>
+			<div class="panel-body">
+				<div class="container-fluid">
+					<p>Zgoščevanje je generiranje neke vrednosti iz poljubnega niza znakov s pomočjo matematične funkcije. Zgoščevalna funkcija pa je funkcija, ki vzame nek vhodni niz in generira drugačen niz, za
+njo pa morajo veljati pravila.
+<ol style="margin-top: 10px;">
+<li>Zgoščevalne funkcijo morajo biti deterministične, kar pomeni, da morajo za določen vhod
+vedno vrniti enak izhod.</li>
+<li>Zgoščevalne funkcije morajo stremeti k temu, da so uniformne, torej da dva različna vhoda ne
+moreta tvoriti enakega izhoda. To je v praksi zelo težko doseči in so ponavadi zadovoljive
```

Comment:
V resnici je ob naslednji predpostavki (izhod fiksne dolžine) to nemogoče doseči. Mogoče bi bilo bolje, če ti predpostavki zamenjaš, pa to pojasniš.